packages/commerce-sdk-react/src/hooks/ShopperBaskets/mut.test.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {act} from '@testing-library/react'
+import {ShopperBasketsTypes, ShopperCustomersTypes} from 'commerce-sdk-isomorphic'
+import nock from 'nock'
+import {
+    assertUpdateQuery,
+    DEFAULT_TEST_CONFIG,
+    mockMutationEndpoints,
+    mockQueryEndpoint,
+    renderHookWithProviders
+} from '../../test-utils'
+import {useCustomerBaskets} from '../ShopperCustomers'
+import {ApiClients, Argument} from '../types'
+import {ShopperBasketsMutation, useShopperBasketsMutation} from './mutation'
+import * as queries from './query'
+
+jest.mock('../../auth/index.ts', () => {
+    return jest.fn().mockImplementation(() => ({
+        ready: jest.fn().mockResolvedValue({access_token: 'access_token'})
+    }))
+})
+
+type Client = ApiClients['shopperBaskets']
+type Basket = ShopperBasketsTypes.Basket
+type BasketsResult = ShopperCustomersTypes.BasketsResult
+
+/** Create an options object for Shopper Baskets endpoints, with `basketId` pre-filled. */
+const makeOptions = <Method extends Exclude<keyof Client, 'clientConfig'>>(
```

Comment:
Nit: `createOptions` sounds better :P

Reply:
Counterpoint, it's 2 more characters! :P